### PR TITLE
Zone space coach reports

### PR DESCRIPTION
**Pull request by rtibbles**
Adds url based zone filtering for coach reports.

### CODE REVIEW

File: kalite/distributed/templates/distributed/base.html [hunk 1]
```diff
@@ -158,7 +158,7 @@
           CENTRAL_SERVER_HOST         : "{{ central_server_host }}",
           SECURESYNC_PROTOCOL         : "{{ securesync_protocol }}",
           CURRENT_LANGUAGE            : "{{ current_language }}",
-          CHANNEL                     : "{{ channel }}",
+          CHANNEL                     : "{{ channel }}"
```
Comment (jamalex) [inline kalite/distributed/templates/distributed/base.html]: Ooooh... has anybody tested 0.14 on IE7 recently? This extra comma, in base.html, would likely break the whole site?   @EdDixon @MCGallaspy 

Do we even support IE7 anymore? Or only IE8?

File: kalite/control_panel/templates/control_panel/partials/_groups_table.html [hunk 2]
```diff
@@ -70,9 +70,9 @@
                                         {% endif %}
                                     </td>
                                     <td>
-                                        {% if group.id %}
+
                                         {# Translators: this is a verb; by clicking this link, the user will be able to coach students. #}
-                                        <a title="{% blocktrans with groupname=group.name %}Coach group {{ groupname }}.{% endblocktrans %}" href="{% url 'coach_reports' %}{{ facility_id }}/{{ group.id }}/">
+                                        <a title="{% blocktrans with groupname=group.name %}Coach group {{ groupname }}.{% endblocktrans %}" href="{% url 'coach_reports' zone_id=zone_id %}{{ facility_id }}/{% if group.id %}{{ group.id }}{% else %}{{ ungrouped_id }}{% endif %}/">
```
Comment (jamalex) [inline kalite/control_panel/templates/control_panel/partials/_groups_table.html]: I see zone_id getting passed into the coach reports context, but does this table get used elsewhere in the control panel? Or did it already have zone_id?

**Comment by jamalex** [comment]
What happens if you go to /coach/ now? Does it redirect?

**Comment by jamalex** [comment]
Looks good (assuming it all works), but I don't see strong incentive to merge this for 0.14, unless we're still trying to support IE7. Should we perhaps just merge to central-develop and develop?

File: kalite/control_panel/templates/control_panel/partials/_groups_table.html (hunk 2 above)
Comment (rtibbles) [inline kalite/control_panel/templates/control_panel/partials/_groups_table.html]: It already had zone_id.

**Comment by rtibbles** [comment]
Closing in favour of merges suggested above.

File: kalite/distributed/templates/distributed/base.html (hunk 1 above)
Comment (MCGallaspy) [inline kalite/distributed/templates/distributed/base.html]: We don't support IE 7 _or_ 8, due to some incompatibility with an external JS library. ([It's PDF.js](https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-support).)

**Pull request closed without merging**